app/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PushNotificationJob.scala

```diff
@@ -39,25 +39,30 @@ class PushNotificationJob @Inject() (
 
   override def name: String = "push-notification-job"
 
-  override def execute(implicit ec: ExecutionContext): Future[Done] =
+  override def execute(implicit ec: ExecutionContext): Future[Done] = {
+    logger.warn(s"[PushNotificationJob] - Started PushNotificationJob")
```

Comment:
We log at this point in the `JobScheduler` so don't know what the value of this is